src/model/documentfragment.js

```diff
@@ -84,6 +84,16 @@ export default class DocumentFragment {
 	}
 
 	/**
+	 * Artificial parent of `DocumentFragment`. Returns `null`. Added for compatibility reasons.
```

Comment:
Compability with other view/model classes, like `Element`. AFAIR it came up at some point. I think that I was checking elements parents until I found something or got `null`. The parent of `DocumentFragment` was `undefined`. I know I could just do `if ( !element.parent ) {` but I think having this unified is better.